draft-ietf-ohai-ohttp.md

```diff
@@ -1014,11 +1015,15 @@ Encapsulated Response.
 ### Differential Treatment {#differential}
 
 A relay MAY add information to requests if the Client is aware of the nature of
-the information that could be added.  The Client does not need to be aware of
-the exact value added for each request, but needs to know the range of possible
-values the relay might use.  Importantly, information added by the relay - beyond
-what is already revealed through encapsulated requests from Clients - can reduce
-the size of the anonymity set of Clients at a gateway.
+the information that could be added.  Any addition MUST NOT include information
+that directly identifies the Client, including any pseudonymous identifier.
```

Comment:
```suggestion
that uniquely and permanently identifies the Client, including any pseudonymous identifier.
```

We've talked about use of identifiers for "buckets" of clients for the purposes of DoS protection. I don't believe in the value they afford, but this text seems like it might prohibit that and prevent use of OHTTP for use cases that would certainly benefit from it. The proposed change tries to clarify that information added shouldn't be unique to a client for a long period of time, as that's basically equivalent to a unique client identifier. But something that changes regularly, or identifies a group of clients, seems reasonable in certain deployments.

Reply:
Right.  This is just a more precise wording either way.

The specific circumstances that might motivate some form of "identification" will require careful consideration on the part of all involved, so we don't need to get too deep into specifics.